qiita_db/support_files/qiita-db-unpatched.sql

```diff
@@ -1891,7 +1891,8 @@ CREATE TABLE qiita.qiita_user (
     receive_processing_job_emails boolean DEFAULT false,
     social_orcid character varying DEFAULT NULL,
     social_researchgate character varying DEFAULT NULL,
-    social_googlescholar character varying DEFAULT NULL
+    social_googlescholar character varying DEFAULT NULL,
```

Comment:
A few requests:

1. Could you undo the changes on this file and add them [as a SQL patch](https://github.com/qiita-spots/qiita/blob/master/CONTRIBUTING.md#developer-workflow)? Do not worry about changing the DBSchema, I'll do that with this and other minor changes it has happened before the release. As a reminder,  the reason those changes were made in this file was because I merged all changes into a single file a few PRs ago. 
2. Could you make the default of creation_timestamp NOW()? That way it will be controlled by the database during creation. 

Thank you.

Reply:
@1 oh, I thought the patching mechanism is dead as I didn't see patch files anymore :-) I've moved the change into `93.sql`. Changes are much cleaner this way.
@2 love to :-)

Reply:
1. in fact, I think you should use 92.sql which is the patch for this release; fine to do that on the other PR if you want me to merge this one first
2. 👍🏽

Reply:
@1: I moved the alterations to `92.sql`. I didn't know that you collect changes for a release. However, tests fail. I might overlook something, but I have the impression that no patch is currently applied in the github actions?!

Reply:
>     2. Could you make the default of creation_timestamp NOW()? That way it will be controlled by the database during creation.

As a follow up: I've just made the DB changes in my "productive" instance and found that all existing users (which did not have the `creation_timestamp` column of cause) now have the timestamp of DB change as their `creation_timestamp` information, even though they existed for many month in the past. 

Is that really the behaviour we want to see in a productive Qiita, @antgonza ?

Reply:
I think that's fine.